src/RequestPage.php

```diff
@@ -326,6 +326,10 @@ function requestHistoryDisplay(AccountRequest &$accountRequest, array &$history,
 		$row .= Html::closeElement('h5');
 		$row .= Html::element('p', [], $historyEntry->comment);
 		$row .= Html::closeElement('div');
+		if ($AccountRequestHistoryEntry->$comment =! null) {
```

Comment:
This doesn't work for a few reasons:
1. You need to modify line 327 (that is the line that includes the comment). In pseudocode, the line could be described this way: "If the comment was made by the user than output a <p> with the comment escaped (i.e. what's already there), and if the comment was made by an admin then run Linker::formatComment on it". What the code you have here does is output *both* the escaped comment *then* the link-formatted comment *after* doing everything it does (so you would see the comment three separate times).
2. The syntax is off here. `!=` is not a valid operator in PHP as far as I'm aware. Also you would want to do `->comment`, not `->$comment` (that's a bit of a quirk in PHP syntax admittedly, but still important to know)
3. $AccountRequestHistoryEntry is not a variable, that's a type (the appropriate variable here is `$historyEntry`)

Just a general comment, please make sure to test your code before submitting it (and ideally before even committing it at all).